server/routes/getValidatorNamesAndAddresses.ts

```diff
@@ -0,0 +1,76 @@
+/* eslint-disable guard-for-in */
```

Comment:
GET requests should NEVER update the DB, please update this route accordingly.

Reply:
Yes Drew I would remove this once when I do all this stuff in eventhandler.ALl data updating would be done thorugh event-handlers.Currently I want data updated in DB so that I can work and test client side pagination and search functionality

Reply:
Events were not triggering our end and we needed data in DB so I just updated the DB through my routes.WOuld remove once get event-handlers and events ready

Reply:
Please update the `chain-events` dependency to `0.2.15` so you can get events working.